examples/ocis-compose/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.4'
+
+services:
+  ocis:
+    depends_on:
+    - redis
+    environment:
+      KONNECTD_IDENTIFIER_REGISTRATION_CONF: /etc/ocis/identifier-registration.yml
+      KONNECTD_ISS: https://localhost:9200
+      KONNECTD_LOG_LEVEL: debug
+      KONNECTD_TLS: '0'
+      OCIS_LOG_LEVEL: debug
+      PHOENIX_OIDC_AUTHORITY: https://localhost:9200
+      PHOENIX_OIDC_METADATA_URL: https://localhost:9200/.well-known/openid-configuration
+      PHOENIX_WEB_CONFIG_SERVER: https://localhost:9200
+      PROXY_HTTP_ADDR: 0.0.0.0:9200
+      REVA_OIDC_ISSUER: https://localhost:9200
```

Comment:
we need to be able to use the service through a public IP address, not only through localhost

e.g. you can use `${OCIS_DOMAIN}` like in https://github.com/owncloud-docker/compose-playground/blob/master/examples/eos-compose-acceptance-tests/docker-compose.yml#L54-L63

and the user will have to set the domain by:
`echo "OCIS_DOMAIN=1.2.3.4" > .env`